app/facades/landmarks_show_facade.rb

```diff
@@ -9,13 +9,42 @@ def landmark
     @_landmark ||= Landmark.find(@id)
   end
 
+  def landmark_category
+    landmark.category.capitalize
+  end
+
   def picture
-    @_picture ||= service.get_picture(landmark.photo_reference)
+    @_picture ||= google_service.get_picture(landmark.photo_reference)
+  end
+
+  def landmark_recordings
```

Comment:
Need to make sure the page still loads even if all functionality isn't built out yet

Reply:
It works!